bin/starter-pack/_main.php

```diff
@@ -14,20 +14,30 @@ function add_extension_register_script() {
 		return;
 	}
 
+	$script_path       = '/build/index.js';
+	$script_asset_path = dirname( __FILE__ ) . '/build/index.asset.php';
+	$script_asset      = file_exists( $script_asset_path )
+		? require( $script_asset_path )
+		: array( 'dependencies' => array(), 'version' => filemtime( $script_path ) );
+	$script_url = plugins_url( $script_path, __FILE__ );
+
 	wp_register_script(
 		'{{extension_slug}}',
-		plugins_url( '/build/index.js', __FILE__ ),
-		array(
-			'wp-hooks',
-			'wp-element',
-			'wp-i18n',
-			'wc-components',
-		),
-		filemtime( dirname( __FILE__ ) . '/build/index.js' ),
+		$script_url,
+		$script_asset['dependencies'],
+		$script_asset['version'],
 		true
 	);
 
+	wp_register_style(
+		'{{extension_slug}}',
+		plugins_url( '/build/style.css', __FILE__ ),
+		array(),
```

Comment:
Might want to add a comment here about ensuring any dependencies the style might have (`wp-components` etc) are manually added here.